src/controllers/DashboardController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Cronbeat\Controllers;
+
+use Cronbeat\Views\DashboardView;
+
+class DashboardController extends BaseController {
+    public function doRouting() {
+        $path = $this->parsePathWithoutController();
+        $pathParts = explode('/', $path);
+        $action = !empty($pathParts[0]) ? $pathParts[0] : 'index';
+        
+        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+            $action = 'process';
+        }
+        
+        switch ($action) {
+            case 'index':
+                $this->showDashboard();
+                break;
+            case 'process':
+                $this->processDashboard();
+                break;
+            default:
+                $this->showDashboard();
+                break;
+        }
```

Comment:
@jetbrains-junie simplify: it all is just showdashboard, so we can just go with "index" action